### DISCUSSION TITLE
Vulnerabilities Detection In Smart Contract

### DISCUSSION THREAD

**Discussion by Saniabuh** [NomicFoundation/hardhat#4034] [General]
Hello Everyone,

My name is Sani Abuh, a Student from University of Bradford. I am analyzing vulnerabilities in smart contract using dynamic approach. I don't much about smart contract code, please can anyone help with dataset and also tools I can use for the analysis. I have tested some code using Remix and solidity compiler but return with error code. I really need assistance as these is part of my dissertation. Thanks

**Comment by fvictorio** [comment]
Hi @Saniabuh , this discussion board is only for questions related to Hardhat.
